frontend/src/component/suggestChanges/SuggestedChangeOverview/SuggestedChangeReviewStatus/SuggestedChangeReviewStatus.tsx

```diff
@@ -30,13 +30,11 @@ export const SuggestedChangeReviewStatus: FC<
                 />
             </StyledButtonContainer>
             <StyledReviewStatusContainer approved={approved}>
-                <StyledFlexAlignCenterBox>
-                    <ConditionallyRender
-                        condition={approved}
-                        show={<Approved approved={approved} />}
-                        elseShow={<ReviewRequired approved={approved} />}
-                    />
-                </StyledFlexAlignCenterBox>
+                <ConditionallyRender
+                    condition={approved}
+                    show={<Approved approved={approved} />}
+                    elseShow={<ReviewRequired approved={approved} />}
```

Comment:
Just with regards to these two components: is there **ever** a case where `Approved.approved`  isn't `true`? And similarly, is there ever a case where `ReviewRequired.approved` is `false`? Just seems a bit redundant to me, but I don't have all the context here 💁🏼

Reply:
Right, thanks for the clarification. But just so I understand fully: `approved` seems to be a boolean. In what cases is a review required if a change request has already been approved? And similarly, when would you render an `Approved` component if `approved` is false?

Reply:
I guess it's poorly named. These two components, ReviewRequired and Approved share some styled components that change the color based on the property they receive. It's not a prerequisite for any logic inside these components, but a way to control how the components inside will display when rendered. 

I will follow up with a PR to make this more clear.